lib/ansible/modules/network/iosxr/iosxr_config.py

```diff
@@ -26,7 +26,7 @@
 module: iosxr_config
 version_added: "2.1"
 author: "Ricardo Carrillo Cruz (@rcarrillocruz)"
-short_description: Manage Cisco IOS XR configuration sections
+short_description: Manage Cisco IOS-XR configuration sections
```

Comment:
Good spot
hum, so it's `NX-OS` http://www.cisco.com/c/en/us/products/ios-nx-os-software/nx-os-software/index.html

But `IOS XR` http://www.cisco.com/c/en/us/support/ios-nx-os-software/ios-xr-software/tsd-products-support-series-home.html